eng/pipelines/templates/jobs/azuresdkpartnerdrops-to-nuget.yml

```diff
@@ -0,0 +1,97 @@
+resources:
+  repositories:
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+      ref: refs/tags/azure-sdk-build-tools_20211215.1
+
+parameters:
+- name: BuildToolsRepoPath
+  type: string
+  default: '$(System.DefaultWorkingDirectory)/_azure-sdk-build-tools'
+- name: ArtifactsPath
+  type: string
+  default: ''
+- name: PartnerDropsBlobPath
+  type: string
+  default: ''
+- name: NugetVersion
+  type: string
+  default: ''
+- name: PartnerDropsSubscription
+  type: string
+  default: ''
+- name: PartnerDropsStorageName
+  type: string
+  default: ''
+- name: PartnerDropsContainerName
+  type: string
+  default: ''
+- name: PartnerDropsBlobPrefix
+  type: string
+  default: ''
+
+jobs:
+  - job: NupkgReleases
+    pool:
+      name: azsdk-pool-mms-win-2019-general
+      vmImage: MMS2019
+
+  steps:
+    - template: tools/clone-buildtools/clone-buildtools.yml@azure-sdk-build-tools
```

Comment:
Now that DevOps supports multiple cloning in a pipeline we should try to use that feature instead of manually cloning ourselves.

Reply:
Do you mean multiple checkout?

Reply:
Yes multiple checkout. Similar to https://github.com/Azure/azure-sdk/blob/main/eng/ci.yml#L20.